gui-wm/cage/cage-0.1.2.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="wayland kiosk"
+HOMEPAGE="https://www.hjdskes.nl/projects/cage/"
+
+if [[ "${PV}" == "9999" ]]; then
```

Comment:
remove the quotes (not needed in `[[ ]]`)

Reply:
always try to use the following idiom:
`if [[ ${PV} == *9999 ]]; then`, because this allows you to have per-branch live ebuilds (like in libreoffice for instance)

Reply:
done